src/test/java/count/ParserTest.java

```diff
@@ -0,0 +1,43 @@
+package count;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import count.exception.IncorrectFormatException;
+import count.exception.InvalidCommandException;
+
+public class ParserTest {
+    String rootFilePath = "./Count.txt";
+    String markIncorrectFormatString = "Use a number after mark/unmark/delete to specify the task targeted!\n" +
+            "Type 'help' to see correct formatting examples";
+    String deadlineIncorrectFormatString = "Invalid format for event or deadline!\n" +
+            "Type 'help' to see correct formatting examples";
+
```

Comment:
Could the following method names be made less lengthy? Because we can already tell what exception should be thrown within the method itself.